backend/controllers/authentication.go

```diff
@@ -0,0 +1,97 @@
+package controllers
+
+import (
+	"backend/config"
+	"net/http"
+
+	"github.com/gin-contrib/sessions"
+	"github.com/gin-gonic/gin"
+)
+
+// login body data definiton
+type Login struct {
+	Username string `form:"username" json:"username" binding:"required"`
```

Comment:
Check with Calvin for login structure as users enter email and password instead of username and password to log in to their account in his PR.

![image](https://github.com/user-attachments/assets/05367ec2-c298-452b-9b48-9c7e28d2e987)

Reply:
Fixed thanks for the notice!